apps/homepage/src/app/_shared/menuButton/menuButton.tsx

```diff
@@ -0,0 +1,15 @@
+'use client'
+import styles from './menuButton.module.scss';
+import Link from 'next/link';
+
+type menuButtonProps = {
+  text:string;
+  href:string;
+}
+
+
+export function MenuButton({ text, href } : menuButtonProps) {
+  return (
+      <Link href={href} className={styles.button}>{text}</Link>
+  )
+} 
```

Comment:
The same thing applies here as above, it would be a better pattern if we used children when we want to pass any element inside the component.